openstack-mixin/README.md

```diff
@@ -4,9 +4,11 @@ The OpenStack mixin is a set of configurable Grafana dashboards and alerts.
 
 The OpenStack mixin contains the following dashboards:
 
-- OpenStack cloud overview
-- OpenStack nova and neutron overview
-- OpenStack cinder and glance overview
+- OpenStack overview
+- OpenStack Nova
+- OpenStack Neutron
+- OpenStack Cinder
+- OpenStack Glance
```

Comment:
Looks like the Glance dashboard got added to the overview dashboard, so we don't need to call it out here :)